.github/workflows/production_pipeline.yml

```diff
@@ -257,3 +259,16 @@ jobs:
           cf set-env $APP_NAME CSV_DOWNLOAD_PAAS_INSTANCE $CSV_DOWNLOAD_PAAS_INSTANCE
           cf set-env $APP_NAME SENTRY_DSN $SENTRY_DSN
           cf push $APP_NAME --strategy rolling
+
+  aws_deploy:
+    name: AWS Deploy
+    if: github.ref == 'refs/heads/main'
+    needs: [lint, test, feature_test, audit]
+    uses: ./.github/workflows/aws_deploy.yml
+    with:
+      aws_account_id: 977287343304
+      aws_resource_prefix: core-prod
+      environment: production
+      release_tag: ${{ needs.test.outputs.releasetag }}
```

Comment:
I felt that passing through the calculated release tag, rather than recalculating it in aws_deploy, was nicer. It might be nicer still to start the production pipeline action with a job which calculates the release tag. All other jobs will depend on this job, so we won't have to keep recalculating the release tag everywhere. I thought better to raise a PR before changing anything more though.